mkosi/machine.py

```diff
@@ -41,36 +42,45 @@ def __init__(self, args: Sequence[str] = [], debug: bool = False) -> None:
         self.debug = debug
         self.stack = contextlib.ExitStack()
         self.args: MkosiArgs
+        self.config: Namespace
+
+        self.config = parse_args(args)["default"]
+
+        # Add fixed options to the machine itself, rather than passing it for every testing function.
+        self.config.force = 1
+        self.config.autologin = True
+        self.config.ephemeral = True
+        if self.config.verb == Verb.qemu:
+            self.config.bootable = True
+            self.config.qemu_headless = True
+            self.config.hostonly_initrd = True
+            self.config.netdev = True
+            self.config.ssh = True
+        elif self.config.verb not in (Verb.shell, Verb.boot):
+            die("No valid verb was entered.")
```

Comment:
huh, I'm not sure where this code came from bc I'm not using `self.config`. Anyway I'll address this on the second PR where we separate build and boot, since for simply merging the CI only one call of `load_args()` is needed.

Reply:
Euhhh my bad this was a comment from a long time ago, feel free to ignore